src/Stock.js

```diff
@@ -100,14 +144,24 @@ class Stock extends React.Component {
                                 marker: { color: 'red' },
                             }
                         ]}
-                        layout={{ width: 800, height: 600, title: `${this.state.graphMode} ${this.state.stockTicker}` }}
-                    /> :
+                        layout={{ width: 800, height: 600, title: `${this.state.graphMode}${this.state.companyName}` }} /><div className="stockInfo">
+
+                            <h4>{this.state.stockTicker} Information:</h4>
+                            <p>Volume {volumeText}: {this.state.stockVolumeValues.at(0)}</p>
+                            <p>52 Week High: {this.state.company_52WeekHigh}</p>
+                            <p>52 Week Low: {this.state.company_52WeekLow}</p>
+                            <p>Profit Margin: {this.state.profitMargin}</p>
+                            <p>Price to Shares Ratio (TTM): {this.state.priceToSalesRatio}</p>
+                            <p>Revenue Per Share (TTM) {this.state.RevenuePerShareTTM}</p>
+
```

Comment:
Can probably get rid of this new line for the same reason as the other, and just make the div closure be right under the stock info.

Reply:
Decided I would just do this change (as I already had it done while checking to see if it would help the flow of the code)